docs/content/documentation/modules/phase_field/Multi Phase/KKS Derivations.md

```diff
@@ -0,0 +1,362 @@
+# KKS Model Definitions
+
+$$
+F = [1-h(\eta)]F_a + h(\eta)F_b+wg(\eta)
+$$
+
+$$
+c =  [1-h(\eta)]c_a + h(\eta)c_b
+$$
+
+$$
+\frac{dF_a}{dc_a} = \frac{dF_b}{dc_b}
+$$
+
+# Materials
+
+The KKS system uses multiple MOOSE materials to provide values for free energy functions, the switching function $h(\eta)$, and the double well function $g(\eta)$.
+Providing these as materials allows the functions to be bundled in a single place, while being used by multiple kernels. Furthermore the automatic differentiation feature
+used in the parsed function materials provides the necessary derivatives at no cost to the developer. The derivatives are stored in material properties and follow a naming scheme
+that is defined in ```KKSBaseMaterial.C```.
+
+# Cahn-Hilliard Kernels
+
+## KKSSplitCHCRes
+This is the split version. In this kernel, we calculate the chemical potential $\mu$ from $\frac{\partial F}{\partial c}$. The non-linear variable for this Kernel is the concentration $c$. To calculate $\frac{\partial c}{\partial t}$ and $\nabla \cdot M(c) \nabla \mu$, we use the `CoupledTimeDerivative` and `SplitCHWRes` kernels, respectively, as described [here](phase_field/Phase Field Equations.md).
```

Comment:
You have references to objects throughout this document, they should probably link to the markdown files.